diff --git a/app/javascript/widget/components/ChatHeader.vue b/app/javascript/widget/components/ChatHeader.vue
index c075222d..30a45bb1 100644
--- a/app/javascript/widget/components/ChatHeader.vue
+++ b/app/javascript/widget/components/ChatHeader.vue
@@ -71,7 +71,7 @@ export default {
     },
     replyWaitMeessage() {
       return this.isOnline
-        ? this.replyTimeStatus
+        ? this.$t('TEAM_AVAILABILITY.ONLINE')
         : this.$t('TEAM_AVAILABILITY.OFFLINE');
     },
   },
