diff --git a/config/initializers/mailer.rb b/config/initializers/mailer.rb
index 54a22f0d..65d1d280 100644
--- a/config/initializers/mailer.rb
+++ b/config/initializers/mailer.rb
@@ -20,7 +20,11 @@ Rails.application.configure do
   smtp_settings[:authentication] = ENV.fetch('SMTP_AUTHENTICATION', 'login').to_sym if ENV['SMTP_AUTHENTICATION'].present?
   smtp_settings[:domain] = ENV['SMTP_DOMAIN'] if ENV['SMTP_DOMAIN'].present?
   smtp_settings[:user_name] = ENV['SMTP_USERNAME']
-  smtp_settings[:password] = ENV['SMTP_PASSWORD']
+  smtp_settings[:password] = if !ENV['SMTP_PASSWORD_FILE'].nil? && File.file?(ENV['SMTP_PASSWORD_FILE'])
+    File.open(ENV['SMTP_PASSWORD_FILE']).read
+   else
+    ENV['SMTP_PASSWORD']
+   end
   smtp_settings[:enable_starttls_auto] = ActiveModel::Type::Boolean.new.cast(ENV.fetch('SMTP_ENABLE_STARTTLS_AUTO', true))
   smtp_settings[:openssl_verify_mode] = ENV['SMTP_OPENSSL_VERIFY_MODE'] if ENV['SMTP_OPENSSL_VERIFY_MODE'].present?
   smtp_settings[:ssl] = ActiveModel::Type::Boolean.new.cast(ENV.fetch('SMTP_SSL', true)) if ENV['SMTP_SSL']
