diff --git a/config/environments/production.rb b/config/environments/production.rb
index fb6c124d..28ebab19 100644
--- a/config/environments/production.rb
+++ b/config/environments/production.rb
@@ -122,9 +122,7 @@ Rails.application.configure do
       origins '*'
       resource '/packs/*', headers: :any, methods: [:get, :options]
       resource '/audio/*', headers: :any, methods: [:get, :options]
-      if ActiveModel::Type::Boolean.new.cast(ENV.fetch('CW_API_ONLY_SERVER', false))
-        resource '*', headers: :any, methods: :any, expose: ['access-token', 'client', 'uid', 'expiry']
-      end
+      resource '*', headers: :any, methods: :any, expose: ['access-token', 'client', 'uid', 'expiry']
     end
   end
 end
