diff --git a/app/javascript/dashboard/assets/scss/super_admin/index.scss b/app/javascript/dashboard/assets/scss/super_admin/index.scss
index 76e2d71a..23710777 100644
--- a/app/javascript/dashboard/assets/scss/super_admin/index.scss
+++ b/app/javascript/dashboard/assets/scss/super_admin/index.scss
@@ -28,8 +28,8 @@
     margin-right: var(--space-one);
   }

-  div {
-    font-size: var(--font-size-small);
+  label {
+    font-size: var(--font-size-small) !important;
     line-height: 1.5;
     margin-bottom: var(--space-normal);
   }
diff --git a/app/views/installation/onboarding/index.html.erb b/app/views/installation/onboarding/index.html.erb
index 8287714c..b58a4ee4 100644
--- a/app/views/installation/onboarding/index.html.erb
+++ b/app/views/installation/onboarding/index.html.erb
@@ -43,9 +43,9 @@
                 </label>
                 <div class="update-subscription--checkbox">
                   <%= check_box_tag "subscribe_to_updates", 'true', true %>
-                  <div for="subscribe_to_updates">
-                  Subscribe to release notes, newsletters & product feedback surveys.
-                  </div>
+                  <label for="subscribe_to_updates">
+                    Subscribe to release notes, newsletters & product feedback surveys.
+                  </label>
                 </div>
                 <button type="submit" class="button nice large expanded">
                   Finish Setup
